docs/wiki/Fetching latest changes - Scenario 1 Module library only.md

```diff
@@ -0,0 +1,99 @@
+When the CARML library is consumed with the [*Template Orchestration*](./Solution%20creation#template-orchestration) approach modules need to be stored in an available location, where the Azure Resource Manager (ARM) can access them. This can be achieved by storing the modules templates in an accessible location like local, Template Specs or the Bicep Registry.
```

Comment:
As mentioned for Scenario 2, I suggest not to map scenarios 1 and 2 with orchestrations and remove this reference to avoid confusion. Modules need to be stored in an available location also if leveraging pipeline orchestration.